docs/deployed-contracts/hoodi.md

```diff
@@ -0,0 +1,118 @@
+- Agent Repo: [`0x7AA5670B2b4f6f0F7369F4F701C03ebFCe97d130`](https://hoodi.cloud.blockscout.com/address/0x7AA5670B2b4f6f0F7369F4F701C03ebFCe97d130) (proxy)
+- Lido App Repo: [`0xd3545AC0286A94970BacC41D3AF676b89606204F`](https://hoodi.cloud.blockscout.com/address/0xd3545AC0286A94970BacC41D3AF676b89606204F) (proxy)
+- Lido Oracle (Legacy Oracle) Repo: [`0x6E0997D68C1930a76413DE7da666D8A531eF1f9b`](https://hoodi.cloud.blockscout.com/address/0x6E0997D68C1930a76413DE7da666D8A531eF1f9b) (proxy)
+- Node Operators Registry Repo: [`0x52eff83071275341ef0A5A2cE48ee818Cef44c39`](https://hoodi.cloud.blockscout.com/address/0x52eff83071275341ef0A5A2cE48ee818Cef44c39) (proxy)
+- Simple DVT Repo: [`0x2b8B52A5e3485853aDccED669B1d0bbF31D40222`](https://hoodi.cloud.blockscout.com/address/0x2b8B52A5e3485853aDccED669B1d0bbF31D40222) (proxy)
+- Sandbox Repo: [`0x89D37eC788988e98BEceB32a8774394F1338B09C`](https://hoodi.cloud.blockscout.com/address/0x89D37eC788988e98BEceB32a8774394F1338B09C) (proxy)
+- EVMScriptRegistry: [`0xe4D32427b1F9b12ab89B142eD3714dCAABB3f38c`](https://hoodi.cloud.blockscout.com/address/0xe4D32427b1F9b12ab89B142eD3714dCAABB3f38c) (proxy)
+- CallsScript: [`0xfB3cB48d81eC8c7f2013a8dc9fA46D2D48112c3A`](https://hoodi.cloud.blockscout.com/address/0xfB3cB48d81eC8c7f2013a8dc9fA46D2D48112c3A)
+- Lido APMRegistry: [`0x15EBf349e1ee9Cd949049fD9352D0c94De046d7b`](https://hoodi.cloud.blockscout.com/address/0x15EBf349e1ee9Cd949049fD9352D0c94De046d7b) (proxy)
+- Aragon APMRegistry: [`0x948ffB5fDA2961C60ED3Eb84c7a31aae42EbEdCC`](https://hoodi.cloud.blockscout.com/address/0x948ffB5fDA2961C60ED3Eb84c7a31aae42EbEdCC) (proxy)
+- Gate Seal Blueprint: [`0x8685Ca0311E4aBd846ee1b5b8B09299E990523F7`](https://hoodi.cloud.blockscout.com/address/0x8685Ca0311E4aBd846ee1b5b8B09299E990523F7)
+- Gate Seal Factory: [`0xA402349F560D45310D301E92B1AA4DeCABe147B3`](https://hoodi.cloud.blockscout.com/address/0xA402349F560D45310D301E92B1AA4DeCABe147B3)
+- Gate Seal: [`0x2168Ea6D948Ab49c3D34c667A7e02F92369F3A9C`](https://hoodi.cloud.blockscout.com/address/0x2168Ea6D948Ab49c3D34c667A7e02F92369F3A9C)
+
+## Data Bus
+
+- DataBus on Chiado (Testnet): [`0x37De961D6bb5865867aDd416be07189D2Dd960e6`](https://gnosis-chiado.blockscout.com/address/0x37De961D6bb5865867aDd416be07189D2Dd960e6)
+
+## Staking modules
+
+### Curated Module
+
+- Node Operators Registry: [`0x5cDbE1590c083b5A2A64427fAA63A7cfDB91FbB5`](https://hoodi.cloud.blockscout.com/address/0x5cDbE1590c083b5A2A64427fAA63A7cfDB91FbB5)
+
+### Simple DVT Module
+
+- Node Operators Registry: [`0x0B5236BECA68004DB89434462DfC3BB074d2c830`](https://hoodi.cloud.blockscout.com/address/0x0B5236BECA68004DB89434462DfC3BB074d2c830)
+
+### Sandbox Module
+
+- Node Operators Registry: [`0x682E94d2630846a503BDeE8b6810DF71C9806891`](https://hoodi.cloud.blockscout.com/address/0x682E94d2630846a503BDeE8b6810DF71C9806891)
```

Comment:
isn't mentioned on the deployed artifacts :thinking:

Reply:
Was deployed separate from scratch deployment. Prepared the PR with fix:
https://github.com/lidofinance/core/pull/988

Reply:
got it